packer_templates/rocky/scripts/cleanup.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/sh -eux
```

Comment:
If these aren't specific to Rocky they should just source the centos files like we do with Alma Linux

Reply:
At the moment these scripts have zero specific to Rocky actually, but who knows maybe sometime they will be extended. Added there as they used in "scripts" section of main JSON file.

Reply:
Actually I can rewrite these includes like you did for springdalelinux.